### PR TITLE
BITMAKER-1828 Improve guide of web application structure

### PR DESCRIPTION
**Pull request by raymond1242**
We currently have a significantly reduced and short description of the Web module for Estela. We should improve it so new devs can better grasp how the project is structured and things are managed.

Completion criteria:
- Provide a more detailed description of the Estela Web module (structure, tools, etc).

### CODE REVIEW

**Comment by raymond1242** [comment]
@mgonnav could you please review this PR, belong ticker : https://tasks.bitmaker.dev/issues/1828?issue_count=5&issue_position=3&next_issue_id=1766&prev_issue_id=1840

File: estela-web/README.md [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+</h1>
 
-This project is created using React Framework (along with Ant Design) and Typescript. This module implements a
-user-friendly environment that communicates with the estela API and lets you manage your spiders and scraping
-projects. For more information on this module and how to set it up, please refer to our
-[official documentation](https://bitmaker.la/docs/bitmaker-cloud/web.html).
+![version](https://img.shields.io/badge/version-0.1-blue)
+![yarn-version](https://img.shields.io/badge/yarn-v1.22.19-blue)
+![build](https://img.shields.io/badge/build-passing-brightgreen)
+![CI](https://github.com/eslint/eslint/workflows/CI/badge.svg)
```
Comment (mgonnav) [inline estela-web/README.md:11]: I think we can get rid of these badges. There is a specific ticket to define this stuff for the whole project.

File: estela-web/README.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
+<p align="center" style="margin-top: 20px">
+  <a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg" alt="react" width="40" height="40"/> </a>
+  <a href="https://nodejs.org/" target="_blank" rel="noreferrer"> <img src="https://www.vectorlogo.zone/logos/nodejs/nodejs-icon.svg" alt="nodejs" width="40" height="40"/> </a>
+</p>
+
+#### The project is created using [React Framework](https://reactjs.org) and [Typescript](https://www.typescriptlang.org). 
+
+This module introduces users with a friendly interface that interacts with the [estela API 🕷](https://github.com/bitmakerla/estela/tree/main/estela-api)  and lets you manage your spiders and scraping projects.
+
+The module allows users to **CREATE**, **UPDATE** and **DELETE** information and data collected from your spiders. Similar to [estela-cli](https://github.com/bitmakerla/estela-cli) but, with an interactive interface.
```
Comment (mgonnav) [inline estela-web/README.md:22]: I think we can be more direct by saying "This module allows users to **manage their scraping projects** and data collected from their spiders. This is a **graphical alternative** to `estela-cli`."

File: estela-web/README.md [hunk 3]
```diff
@@ -1,6 +1,36 @@
...
+    <img width="20" src="https://gw.alipayobjects.com/zos/rmsportal/KDpgvguMpGfqaHPjicRK.svg">
+    </a></span>[**AntDesign**](https://ant.design) A Design Systems Tool that allows to create efficient and enjoyable components to improve the user expereience
+
+- <span style="vertical-align: middle"><a href="https://openapi-generator.tech">
+    <img width="20" src="https://user-images.githubusercontent.com/109659/40094839-2bc8f2ee-5897-11e8-8092-583c26e4d0df.png">
+    </a></span>[**OpenApi**](https://openapi-generator.tech) A Tool that can easily generate documentation from metadata
+
+- <span style="vertical-align: middle"><a href="https://www.typescriptlang.org">
+    <img width="20" src="https://upload.wikimedia.org/wikipedia/commons/4/4c/Typescript_logo_2020.svg">
+    </a></span>[**TypeScript**](https://www.typescriptlang.org) A Strongly **Typed** Programming Language
```
Comment (mgonnav) [inline estela-web/README.md:34]: I think it should suffice to mention these and not give them so much space here. These are just the tools we use.

**Review by mgonnav — CHANGES_REQUESTED** [review]
We should provide a more detailed description of the project. Please provide a description of **how the project is structured and things are managed** (i.e., talk about how we manage components, assets, routes, services, etc.). It might be good to see how other projects detail this.

These changes should be added to the docs and have something more general in this README. Take the time needed to understand and give a detailed description of the web project.

**Comment by raymond1242** [comment]
@mgonnav could you please check this last commit

**Review by mgonnav — APPROVED** [review]
I've proofread the docs and made some fixes. I think we can merge this 👍🏻

**Pull request merged**